code/lib/store/src/csf/composeConfigs.ts

```diff
@@ -53,6 +54,6 @@ export function composeConfigs<TFramework extends AnyFramework>(
     render: getSingletonField(moduleExportList, 'render'),
     renderToDOM: getSingletonField(moduleExportList, 'renderToDOM'),
     applyDecorators: getSingletonField(moduleExportList, 'applyDecorators'),
-    runStep: composeStepRunners<TFramework>(getArrayField(moduleExportList, 'runStep')),
```

Comment:
Clearly my tests here were bad, can you add one that failed with the old code?

Reply:
Done

Reply:
Thank you!